pkg/config/agent/sysadvisor/qosaware/qos_aware_plugin.go

```diff
@@ -31,16 +31,18 @@ type QoSAwarePluginConfiguration struct {
 
 	*resource.ResourceAdvisorConfiguration
 	*server.QRMServerConfiguration
-	*reporter.HeadroomReporterConfiguration
+	*reporter.ReporterConfiguration
+	*reporter.NodeMetricReporterConfiguration
 	*model.ModelConfiguration
 }
 
 // NewQoSAwarePluginConfiguration creates a new qos aware plugin configuration.
 func NewQoSAwarePluginConfiguration() *QoSAwarePluginConfiguration {
 	return &QoSAwarePluginConfiguration{
-		ResourceAdvisorConfiguration:  resource.NewResourceAdvisorConfiguration(),
-		QRMServerConfiguration:        server.NewQRMServerConfiguration(),
-		HeadroomReporterConfiguration: reporter.NewHeadroomReporterConfiguration(),
-		ModelConfiguration:            model.NewModelConfiguration(),
+		ResourceAdvisorConfiguration:    resource.NewResourceAdvisorConfiguration(),
+		QRMServerConfiguration:          server.NewQRMServerConfiguration(),
+		ReporterConfiguration:           reporter.NewReporterConfiguration(),
+		NodeMetricReporterConfiguration: reporter.NewNodeMetricReporterConfiguration(),
```

Comment:
`NodeMetricReporterConfiguration` should be initialized in `NewReporterConfiguration` rather than here

Reply:
NodeMetricReporterConfiguration is included in ReporterConfiguration, so NewNodeMetricReporterConfiguration should be removed here.